assets/scripts/segments/__tests__/__snapshots__/SegmentForPalette.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SegmentForPalette renders width correctly depending on the dimension 1`] = `
+<div
+  className="segment segment-in-palette"
+  onPointerOver={[Function]}
+  style={
+    Object {
+      "width": "NaNpx",
```

Comment:
Is there a way to prevent this from having a `NaN` value? (Or does it not matter for the purpose of the test?)

Reply:
It does not really matter for the tests, since these are snapshots and will not change unless the markup changes. 
we could make these more realistic by providing better values. If you want I can do that.

Reply:
If it doesn't add too much complexity, go for it-- but if it's going to make the test too complicated then we can just leave this alone.